experimental/generation/composer-form-dialog.md

```diff
@@ -0,0 +1,65 @@
+5. **Create a new bot** or **open an existing bot** from the **Home** page. The Forms section will not be enabled if you do not have a bot open.
+
+
+
+## Creating and using a form dialog
+
+The following steps describe how to create a form dialog, how to connect it to your root dialog, and how to re-generate your dialog to reflect changes to your form fields.
+
+
+1. **Create a form**
+The property schema defines the fields your bot needs to fill and values that are acceptable. Values can be defined by an enumerated list or by a type, such as age, email, geography, or number.
+
+    1. In the menu, select the **Forms** icon. You will see this page:    
+    2. Under **Schemas**, select the plus button to add a new schema. 
+    3. In the **"Create a form dialog"** window, provide a name. This will be the name of your schema as well as the dialog generated from it.
+    4. Under **Schemas**, select the schema you just created. The schema editor should appear on the right. 
+    5. Select **Add property**.
+    6. A new property card will appear. Provide a **name** and a **type**.
+    7. **Drag and drop cards to change priority or to make a property optional.** By default, properties are set as `required` and prioritized by they order in the form. ***Required properties*** are properties that your bot will ask the user to provide. The ***priority*** refers to the order in which the bot will ask for the required properties. ***Optional properties*** are properties the bot accepts if given but does not ask for.
+
+3. **Generate your dialog.** When your schema is complete, press the "Generate dialog" button. 
+4. **Inspect your new form dialog**. You can navigate to your dialog by selecting the overflow menu (...) next to the name of the form. You can also use select the **Design** iconin the main menu to see your list of dialogs to view the form dialog. The form dialog will appear in this list. The triggers for the dialog will be grouped by property along with a group for form-wide triggers.
+
+5. **Connect your form dialog to the root dialog.**
+You connect your form dialog to your main dialog as you would any child dialog. For example, if you'd like to invoke your form dialog from the root, you can add a new trigger:
+    1. From your root dialog, select the **Add** dropdown and select the **Add a new trigger**.
+    2. In the **"Create a trigger"** window, select the trigger type **"Intent recognized"**.
+   
+       If your root dialog is using the default recognizer, add example trigger phrases. Add a condition `=turn.recognized.score > 0.8` to prevent your form from restarting.
+    3. In your new trigger, add the action node **Manage properties > Begin a new dialog** and select your form dialog.
+
+Alternatively, you can add a **Begin a new dialog** node as an action in an existing trigger.
```

Comment:
I would simplify this and indicate you can just call begin dialog to invoke the form or alternatively create a trigger.  For creating a trigger might be better to describe how you could use the whole form model.